axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveLineService.java

```diff
@@ -392,6 +392,18 @@ public MoveLine createMoveLine(
             "Traitement de la ligne de facture : compte comptable = {}, montant = {}",
             new Object[] {account.getName(), companyExTaxTotal});
 
+        if (appAccountService.getAppAccount().getAnalyticDistributionRequiredOnInvoiceLines()
+            && invoiceLine.getAnalyticDistributionTemplate() == null
+            && invoiceLine.getAnalyticMoveLineList().isEmpty()
+            && invoiceLine.getAccount().getAnalyticDistributionRequiredOnInvoiceLines()) {
```

Comment:
use account.getAnalyticDistributionRequiredOnInvoiceLines as account is already assign to one local variable.